cypress/fixtures/publisher_articles_after_publish.json

```diff
@@ -0,0 +1,19 @@
+{
+  "articles": [
+    {
+    "id": 2,
+    "title": "Test2",
+    "body": "TestBody2"
+    },
+    {
+    "id": 3,
+    "title": "Test3",
+    "body": "TestBody3"
+    },
+    {
+    "id": 4,
+    "title": "Test4",
+    "body": "TestBody4"
+    }
+  ]
+}
```

Comment:
Please format this json object. For readability....

Reply:
formatted